h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -137,16 +140,45 @@ class SearchBarController extends Controller {
       }
     };
 
+    var lozengify = content => {
+      var lozengeEl = LozengeHelper.createLozenge(content);
+      this._lozenges.appendChild(lozengeEl);
+      new LozengeController(lozengeEl);
```

Comment:
this can be done in the createLozenge function - so that searchbarcontroller does not need to know about it